### PR TITLE
Create Queue using 2 stacks

### PR DESCRIPTION
**Pull request by surajx24**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by surajx24** [comment]
this algorithm was not present so i thought contributing it. quite simple algo but useful.

**Comment by TheSuperNoob** [comment]
This appears to be a direct copy paste of the code on this page: [https://www.geeksforgeeks.org/queue-using-stacks/](url) in the python implementation. Except its missing the credit comment at the bottom of the file.

File: data_structures/queue/Queue using 2 stacks [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  
+        # Push everything back to s1  
+        while len(self.s2) != 0:  
+            self.s1.append(self.s2[-1])  
+            self.s2.pop() 
+  
+    # Dequeue an item from the queue  
+    def deQueue(self): 
+          
+            # if first stack is empty
```
Comment (cclauss) [inline data_structures/queue/Queue using 2 stacks:27]:
```suggestion  
        # if first stack is empty  
```

File: data_structures/queue/Queue using 2 stacks [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        # Push everything back to s1  
+        while len(self.s2) != 0:  
+            self.s1.append(self.s2[-1])  
+            self.s2.pop() 
+  
+    # Dequeue an item from the queue  
+    def deQueue(self): 
+          
+            # if first stack is empty  
+        if len(self.s1) == 0:
```
Comment (cclauss) [inline data_structures/queue/Queue using 2 stacks:28]:
```suggestion
        if not self.s1:  
```

File: data_structures/queue/Queue using 2 stacks [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# Driver code  
+if __name__ == '__main__': 
+    q = Queue() 
+    q.enQueue(1)  
+    q.enQueue(2)  
+    q.enQueue(3)  
+  
+    print(q.deQueue()) 
+    print(q.deQueue()) 
+    print(q.deQueue())
```
Comment (cclauss) [inline data_structures/queue/Queue using 2 stacks:45]:
```suggestion
    queue = Queue() 
    queue.enQueue(1)  
    queue.enQueue(2)  
    queue.enQueue(3)  
  
    print(queue.deQueue()) 
    print(queue.deQueue()) 
    print(queue.deQueue()) 
```

File: data_structures/queue/Queue using 2 stacks [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            self.s2.append(self.s1[-1])  
+            self.s1.pop() 
+  
+        # Push item into self.s1  
+        self.s1.append(x)  
+  
+        # Push everything back to s1  
+        while len(self.s2) != 0:  
+            self.s1.append(self.s2[-1])  
+            self.s2.pop()
```
Comment (cclauss) [inline data_structures/queue/Queue using 2 stacks:22]:
```suggestion
        while self.s2:  
            self.s1.append(self.s2.pop())  
```

File: data_structures/queue/Queue using 2 stacks [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    def __init__(self): 
+        self.s1 = [] 
+        self.s2 = [] 
+  
+    def enQueue(self, x): 
+          
+        # Move all elements from s1 to s2  
+        while len(self.s1) != 0:  
+            self.s2.append(self.s1[-1])  
+            self.s1.pop()
```
Comment (cclauss) [inline data_structures/queue/Queue using 2 stacks:14]:
```suggestion
        while self.s1:  
            self.s2.append(self.s1.pop())
```

File: data_structures/queue/Queue using 2 stacks [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        # Push item into self.s1  
+        self.s1.append(x)  
+  
+        # Push everything back to s1  
+        while len(self.s2) != 0:  
+            self.s1.append(self.s2[-1])  
+            self.s2.pop() 
+  
+    # Dequeue an item from the queue  
+    def deQueue(self):
```
Comment (cclauss) [inline data_structures/queue/Queue using 2 stacks:25]:
```suggestion
    def dequeue(self): 
```
As discussed in CONTRIBUTING.md.

File: data_structures/queue/Queue using 2 stacks [hunk 7]
```diff
@@ -0,0 +1,45 @@
+# Python3 program to implement Queue using  
+# two stacks with costly enQueue()  
+  
+class Queue:  
+    def __init__(self): 
+        self.s1 = [] 
+        self.s2 = [] 
+  
+    def enQueue(self, x):
```
Comment (cclauss) [inline data_structures/queue/Queue using 2 stacks:9]:
```suggestion
    def enqueue(self, x): 
```
Python function names as discussed in CONTRIBUTING.md.

**Comment by cclauss** [comment]
Closing as plagiarized.

**Pull request closed without merging**